src/index.js

```diff
@@ -1 +1,149 @@
 import header from './components/header';
+
```

Comment:
Let's maybe put this in a separate file and import it into index? Any ideas on where to put it @kissyourmisery?